imap_processing/tests/conftest.py

```diff
@@ -46,45 +46,48 @@ def _autoclear_spice():
 
 
 @pytest.fixture(scope="session")
-def _download_de440s(spice_test_data_path):
-    """This fixture downloads the de440s.bsp kernel into the
-    tests/spice/test_data directory if it does not already exist there. The
+def _download_external_kernels(spice_test_data_path):
```

Comment:
Do we need these for the tests? It is possible that we don't and this change isn't necessary.

Reply:
Yep. Ok then leave it.

Reply:
Thanks for chiming in here Tim!